content/en/blog/general/lets-play-kyverno/index.md

```diff
@@ -0,0 +1,95 @@
+---
+date: 2023-06-02
+title: "Let's Play Kyverno"
+linkTitle: "Let's Play Kyverno"
+description: "Away with the hurdles. How to get started with a Kubernetes Native Tool without an actual Kubernetes cluster?"
+---
+
+## Foreword
+
+<i>"Kyverno is a policy engine designed specifically for Kubernetes."</i>
+
+While this approach makes it very easy to use Kyverno in its intended environment, it is sometimes difficult to explain and present the capabilities when that environment is not available.
+
+To help potential users get started and reduce the effort required to test Kyverno and/or develop new policies, the [Kyverno Playground] (https://playground.kyverno.io) has been developed. This has now been available for 3 weeks, currently in version 0.3.1.
```

Comment:
The link here isn't rendering properly. Check that you don't have a space between the link text and the link itself.